advisor-schema/src/main/resources/db/migration/V1__init.sql

```diff
@@ -0,0 +1,95 @@
+--CREATE SCHEMA IF NOT EXISTS advisory_schema;
+--USE SCHEMA 'advisory_schema';
+
+CREATE TYPE user_type AS ENUM (
+    'applicant',
+    'advisor'
+    );
+
+CREATE TYPE advisors_role AS ENUM (
+    'associate',
+    'partner',
+    'senior'
+    );
+
+CREATE TYPE phone_number_type AS ENUM (
+    'home',
+    'work',
+    'mobile'
+    );
+
+CREATE TYPE application_status AS ENUM (
+    'new',
+    'assigned',
+    'on_hold',
+    'approved',
+    'canceled',
+    'declined'
+    );
+
+CREATE TABLE IF NOT EXISTS users
+(
+    id       BIGSERIAL CONSTRAINT users_PK PRIMARY KEY,
+    username VARCHAR(255) NOT NULL CONSTRAINT users_username_UK UNIQUE,
+    email    VARCHAR(255) NOT NULL CONSTRAINT users_email_UK UNIQUE,
+    "type"   user_type    NOT NULL,
+    version                    BIGINT                   NOT NULL DEFAULT 0,
+    CONSTRAINT email_format_check CHECK (email ~* '^[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,}$')
+);
+
+CREATE TABLE IF NOT EXISTS advisors
```

Comment:
why do you use "IF NOT EXISTS" together with flyway?

Reply:
When we add new migrations, Flyway can detect the changes and execute the latest ones. However, it becomes problematic when the version number of the latest script isn’t the highest. In other words, it’s out of order. We can use the outOfOrder configuration parameter to tell Flyway to run these scripts instead of skipping them. Another thing, that I can start this application with predefined image, where this table had been created